tests/core/pyspec/eth2spec/test/bellatrix/block_processing/test_process_execution_payload.py

```diff
@@ -36,8 +36,8 @@ def run_execution_payload_processing(
     If ``valid == False``, run expecting ``AssertionError``
     """
     # Before Deneb, only `body.execution_payload` matters. `BeaconBlockBody` is just a wrapper.
-    # after EIP-7732 the execution payload is no longer in the body
-    if is_post_eip7732(spec):
+    # After Gloas the execution payload is no longer in the body
+    if is_post_gloas(spec):
```

Comment:
I haven't read this entire file but only some of tests in this file call this helper function. It feels like there should be a cleaner way to modify these Bellatrix tests without introducing these invasive `is_post_gloas` checks.

Reply:
Yes, but this is a bit out-of-scope of this PR.

Reply:
Agreed!